packages/web-components/src/components/rux-tabs/rux-tabs.tsx

```diff
@@ -129,8 +130,23 @@ export class RuxTabs {
         this._addTabs()
     }
 
+    componentWillUpdate() {
+        this._checkSelected()
+    }
+
     private _addTabs() {
-        this._tabs = Array.from(this.el.querySelectorAll('rux-tab'))
+        this._tabs = Array.from(this.el?.querySelectorAll('rux-tab'))
+    }
+
+    private _checkSelected() {
+        // If no selected tab exists, we need to set an item to tabindex = 0
+        // so that we can still tab into the tabs list
+        const firstTab: HTMLDivElement | null = this._tabs[0].shadowRoot!.querySelector(
+            '[part="container"]'
```

Comment:
Can you write a test to cover this? only because its so coupled to another component's part name. If we ever remove or rename it, I wouldn't remember to check here and "container" is a part used everywhere so find/replace wouldn't work.

Reply:
Added a new tabs only test and a changes. I'm thinking this is just a patch because it only. enhances existing keyboard functionality it doesn't remove anything old.. but if I need to update the changes let me know!